generated/installer/ibm-spectrum-scale-csi-operator.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: ibm-spectrum-scale-csi-operator
       containers:
       - name: operator
-        image: cp.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-operator@sha256:c93ba727ea55230733cb3d0b9326c0d6a7c6d88667948213b3ca198955bca23f
+        image: cp.stg.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-operator@sha256:0ab8ff664b8c9bca0c240b07bd919384b016aa3d12895169739025002bebb4de
```

Comment:
we can use cp.icr.io only rather cp.srg.icr.io

Reply:
Images are pushed to to staging repo - cp.stg.icr.io